hack/release-tools/tag-release.sh

```diff
@@ -28,9 +28,19 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 function tag_and_push() {
     echo "Tagging and pushing $VELERO_VERSION"
     git tag $VELERO_VERSION
-    git push $VELERO_VERSION
+    
+    if [[ $publish == "TRUE" ]]; then
+        git push upstream $VELERO_VERSION
+    fi
 }
 
+# Default to a dry-run mode
+publish=FALSE
+if [[ "$1" = "publish" ]];
+then
```

Comment:
this can go on line 39.
Staying consistent with other `if [[ ... ]]` in the file

Reply:
same w/ line 83

Reply:
👍  Changed locally, also gonna update docs to reflect the new argument before pushing.